setup.py

```diff
@@ -23,6 +23,7 @@
         "microcosm>=2.12.0",
         "microcosm-daemon>=1.0.0",
         "microcosm-logging>=1.3.0",
+        "pymemcache>=2.2.2",
```

Comment:
This is not to say that we're necessarily tied to it, but we've been using [python-memcached](https://pypi.org/project/python-memcached/) in some of our other internal services. @pelegdvir would probably know better why (iirc, there may have been some discrepancies between how it decided consistent hashing strategies with our JS strategies). This is probably not relevant to a python-only implementation, but just putting it out there.

Reply:
This could probably also go in under `extras` instead. You shouldn't need to worry about caching by default.

Reply:
@GabeLambda good points.

Re #1 - that library seems unmaintained (last update in 2017). The one i'm using is pure python so very lightweight and seems well maintained (last updated few days ago, and maintained by pinterest). I'd be curious to give it a go and if works well can switch the other one around. WDYT?

Re #2 - crossed my mind too, I'll make this change.

Reply:
Yep. We've used the other lib for consistent JS sync (JS memcache libs are awful)